src/main/java/seedu/duke/ui/StringLib.java

```diff
@@ -158,4 +158,24 @@ public interface StringLib {
     String[] FORBIDDEN_CHARS = {"!","@","#","$","%","&","*","(",")","+",",",
                                 ".","/",":",";","<","=",">","?","[","]","^","_","`","{","|","}"};
     String SAVE_SEPARATOR = "#######";
+    String INVALID_ADD_TO_RECIPE_DESCRIPTION = "OOPS!!!\n" +
+            "Looks like the description of the command is invalid!\n" +
+            "To add elements to the recipe, please follow the following layout:\n" +
```

Comment:
Perhaps consider `format` instead of `layout`? 
```suggestion
            "To add elements to the recipe, please follow the following format:\n" +
```

Reply:
Rectified at comment 2. See above.